src/common/reducers/snaps.js

```diff
@@ -51,17 +51,17 @@ export function snaps(state = {
         isFetching: false,
         success: true,
         snaps: [
-          ...action.payload.snaps
+          ...action.payload.response.payload.snaps
```

Comment:
Nested in a bit ugly way, but let's look into that later when we refactor snaps store.